pipeline/03-evaluate.R

```diff
@@ -74,21 +75,10 @@ gen_agg_stats <- function(data, truth, estimate, bldg_sqft,
     prd_no_sop = ~ ifelse(sum(!is.na(.y)) > 1, prd(.x, .y, na.rm = TRUE), NA),
     prb_no_sop = ~ ifelse(sum(!is.na(.y)) > 1, prb(.x, .y, na.rm = TRUE), NA),
     mki_no_sop = ~ ifelse(sum(!is.na(.y)) > 1, mki(.x, .y, na.rm = TRUE), NA),
-    cod = ~ ifelse(
-      sum(!is.na(.y)) > 33,
-      list(ccao_cod(.x / .y, na.rm = TRUE)),
-      NA
-    ),
-    prd = ~ ifelse(
-      sum(!is.na(.y)) > 33,
-      list(ccao_prd(.x, .y, na.rm = TRUE)),
-      NA
-    ),
-    prb = ~ ifelse(
-      sum(!is.na(.y)) > 33,
-      list(ccao_prb(.x, .y, na.rm = TRUE)),
-      NA
-    )
+    cod = ~ ifelse(sum(!is.na(.y)) >= min_n, cod(.x / .y, na.rm = TRUE), NA),
+    prd = ~ ifelse(sum(!is.na(.y)) >= min_n, prd(.x, .y, na.rm = TRUE), NA),
+    prb = ~ ifelse(sum(!is.na(.y)) >= min_n, prb(.x, .y, na.rm = TRUE), NA),
+    mki = ~ ifelse(sum(!is.na(.y)) >= min_n, mki(.x, .y, na.rm = TRUE), NA)
```

Comment:
The only difference between the `_no_sop` and normal version of these stats will now be the minimum number of required sales.